packages/components/src/styles/footnotes/footnotes.stories.ts

```diff
@@ -55,39 +52,42 @@ export const Default = {
 };
 
 /**
- * Use the class with a `ul` element to display the footnotes as an unordered list.
+ * Use the alternative html elements for alternative appearances:
```

Comment:
We are repeting `alternative`, maybe `use the standard html list elements...` or something similar would be a better option. What do you think @coraliefeil @MartaPintoTeixeira?

Reply:
agree

Reply:
Updated